### ISSUE TITLE
[r1.5] Crash with 1.10.2 versions of Baubles

### ISSUE THREAD

**Issue by SPMoo66** [yrsegal/NaturalPledge#10]
Crashes with the newest version of Baubles [1.3.BETA5]

http://pastebin.com/A8nrB0qs

https://github.com/Azanor/Baubles/issues/143

On another note, any chance you could add source or issues tabs for the Curseforge page?

**Comment by SDUBZ** [comment]
did yours also crash on world load?

**Comment by SDUBZ** [comment]
@yrsegal would be nice to hear when your not busy about this problem

**Comment by UnNinja** [comment]
This is still an issue as of baubles beta9. And botania is now requiring the baubles beta (previously it didn't work with them)
[Crash log](http://pastebin.com/VTSSdUuu)

**Comment by yrsegal** [comment]
NP is on hold for 1 or 2 more versions of botania as I plan features.

**Comment by UnNinja** [comment]
New features would be great, but so would actually being able to use the mod. Any chance of a fix for this crash while you're still brainstorming new features? I'd love to continue to use natural pledge, but alas not being able to update botania is going to cause more problems on our server than losing NP

**Comment by yrsegal** [comment]
It should only be a week at the most before the next update hits.

**Comment by 0Navis0** [comment]
I get kinda the same crash at the world loading http://pastebin.com/JaqEr1Ng

**Comment by topher2010** [comment]
I hate to do this, but do we have an update on an update?  My pack updating is waiting on this one mod.

**Comment by yrsegal** [comment]
probably by next week. I want to let Botania stabilize first.

**Comment by topher2010** [comment]
it was Baubles that caused the crashing issue, not Botania.  But ok, i'll have to remove your mod from my pack as i can't update it without either removing your mod or updating your mod.

Thank you for your time.

**Comment by yrsegal** [comment]
Fixed, will be in next release
